core/find_apk.sh

```diff
@@ -8,7 +8,15 @@ getlatestapk() {
   tempapks=( )
   OLDIFS="$IFS"
   IFS="
-" # We set IFS to newline here so that spaces can survive the for loop
+" # We set IFS to newline here so that filenames containing spaces are
+  # handled correctly in the for-loops below.
+
+  # decompress apks
+  # some apks are lz compressed to workaround github file-size limits.
+  for foundapklz in $(find "$1" -iname '*.apk.lz'); do
+    lunzip $foundapklz
```

Comment:
Lets do something like
```
lunzip --keep "$foundapklz"
```

To avoid getting a "deleted" entry in git for the arm64 repo. That can lead to confusion. Also lunzip is no-op if the file is already extracted, so that'll be fine.

For cleaning up, maybe do something like:
```
for f in $(find -iname '*.apk'); do test -e $f.lz || rm $f; done     
```